.github/workflows/deploy.yml

```diff
@@ -0,0 +1,21 @@
+name: 'ecma-262'
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    name: 'deploy github pages'
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'tc39/ecma262' }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ljharb/actions/node/install@56f291b2a0d9e3bce72634356ae949b1e054b9f8
+        name: 'nvm install lts/* && npm ci'
+        with:
+          node-version: lts/*
+          use-npm-ci: true
+      - run: npm run build-only ./scripts/auto-deploy.sh
```

Comment:
Does `npm run` allow you to specify multiple commands, or is this some other syntax I don't recognize?

Reply:
whoops, it is a typo. good catch! saves us a failed master build :-)